include/jwt-cpp/jwt.h

```diff
@@ -845,6 +847,74 @@ namespace jwt {
 			return pkey;
 		}
 
+		inline void freem(RSA*){}
```

Comment:
left of test code? this seems out of place

Reply:
This was a temporary workaround to full the auto pointers